packages/theme/src/theme/_buttons.scss

```diff
@@ -76,6 +76,43 @@
 		padding: 0 $padding-smaller;
 	}
 
+	&-active-icon {
+		background-color: unset;
```

Comment:
We should not force the background. If user want to set one of the defined state as default, and activation change it, it won't be possible with that.